src/components/record/RecordForm.jsx

```diff
@@ -105,10 +114,10 @@ class RecordForm extends React.Component {
     return (
       <>
         <PromiseTrackingMask area="sform" />
-        {!!this.state.form && (
+        {!!this.props.form && (
```

Comment:
Hmmm .... it seems like quite a significant change. I hope it will work as before:)  (CANNOT EASILY JUDGE THAT)

Can you somehow comment on it why you decided to switch?

Reply:
Ok thanks for the info.

Oh, really, the tests are disabled? I did not know. So please make it as highest priority task after you finish all that you started already.

Reply:
oh shoooot ..... forget about it ... I thought we are talking about SForms

Reply:
I estimate: the tests here will be enabled tmw

Reply:
Yes, I am also confused between s-forms and record-manager-ui :')